index.html

```diff
@@ -49,5 +47,183 @@ <h1 class="intro__heading">
+                <li class="project__info--title">canopy</li>
+              <li class="project__info--dot">
+                    <i class="fas fa-circle color__circle"></i>
+                </li>
+                <li class="project__info--author">Back End Dev</li>
+                 <li class="project__info--dot">
+                  <i class="fas fa-circle color__circle"></i>
+                 </li>
+                <li class="project__info--date">2015</li>
+              </ul>
+            <p class="project__desc--text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="list__lng--btn">
+                <li class="lng__btn--cont"> <button type="button" class="lng__btn">html</button></li>
+                <li class="lng__btn--cont"> <button type="button" class="lng__btn">css</button></li>
+                <li class="lng__btn--cont"> <button type="button" class="lng__btn">javascript</button></li>
+            </ul>
+             <button type="button" class="link__btn--project">See Project</button>
+             </figcaption>
+        </figure>
+
+    </section>
+    
+    <section class="about">
+     <div class="intro intro__resume">
+            <h1 class="intro__heading">
+                About Myself
+            </h1>
+            <p class="intro__text">
+                I’m a software developer! I can help you build a product , 
+                feature or website Look through some of my work and experience! If you like what you 
+                see and have a project you need coded, don’t hestiate to contact me.</p>
+             <a href="#" class="intro__contact">let's connect</a>
+            <ul class="intro__social--cont">
+                
+                <li class="social__item"> <a href="#" class="link__social">  <i class="fab fa-github-square"></i> </a> </li>
+                <li class="social__item"> <a href="#" class="link__social">  <i class="fab fa-linkedin-in"></i> </a> </li>
+                <li class="social__item"> <a href="#" class="link__social">  <i class="far fa-hand-peace"></i> </a> </li>
+                <li class="social__item"> <a href="#" class="link__social">  <i class="fab fa-twitter"></i> </a>   </li>
+                <li class="social__item"> <a href="#" class="link__social">  <i class="fab fa-medium-m"></i>   </a>   </li>
+
+            </ul>
+             
+            <button type="button" class="link__btn--resume">Get my resume</button>
+        </div>
+       <div class="skills__cont">
+           <ul class="list__skills--cont">
+               <li class="cont__skill">
+                   <p class="name__skill">Languages</p>
+                   <i class="fas fa-chevron-down fa-xs icon__chevron"></i>
+               </li>
+             
+                    <li class="language__cont">
+                        <figure class="img__cont">
+                            <img src="./images/js.png" alt="" class="img__lang">
```

Comment:
- [ ] For UX and SEO optimization reasons, please add descriptive text to your image alt attributes. Do this as well in other parts of your code where you have `img` tags.

**UX** - Because the text is displayed in the event the image is unable to load. The user can have an idea of what the image describes. It also helps users who are visually impaired and using screen-readers.
**SEO Optimization** - Because it helps search engines better crawl and rank your website. Refer to [this article](https://blog.hubspot.com/marketing/image-alt-text) for more reading, if you wish.